_data/projects/Arachnida.yml

```diff
@@ -0,0 +1,17 @@
+name: Arachnida
+desc: Simple web interface to pilot web-crawlers
+site: https://github.com/guillim/Arachnida
+tags:
+- Javascript
+- Meteor
+- crawler
+- scrapping
+- scrapper
+- upforgrabs
+- contributor
+- opensource
+- js
+- Headlesschrome
+upforgrabs:
+  name: Help wanted
+  link: https://github.com/guillim/Arachnida
```

Comment:
This link should point the curated issues, e.g. `https://github.com/guillim/Arachnida/labels/help%20wanted`

Are you able to update that?

Reply:
Shit I deleted the Forked repo already (stupid me). I am going to have to do a new Pull Request to get these changes right

Reply:
@guillim I am making the desired changes for you then, hope that's fine